tools/create-dns-pka.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+## Unsure why this offset is needed and present in make-dns-cert
+TOTAL_LENGTH=6
+
+function usage {
+        echo "Usage: "$0" -k [fingerprint of a key in your keyring] -d [dns record you want to create] -u [url where to fetch the key from]"
+        echo "Example: create-dns-pka.sh -f 0BE0E990E02DF48CE0A1F199530843F3420AC0D3 -d bobdomain.com -u https://bobdomain.com/gpg/bobkey.pub"
+	echo "Will return : mzbi7kpidyqxg7gzxmpyf7b8n52fmm1n._pka   TYPE37  \# 67 0006 0000 00 14 0CE0E980E01DF48CE0B1F199540843F3420AD0D3 68747470733a2f2f3230303031332e6e65742f6770672f6c756340323030
+4031332e6e65742e707562 "
+        echo '   -f   fingerprint of a gpg key already imported in your local keyring.'
+	echo '   -d   the domain name where you want to publish this record'
```

Comment:
What do you need this information for?
In line 23 you are assigning it to the variable `RR`, which is never used beside the `if` in line 34....

Reply:
correct. I wrote this a while back and I don't recall why I added the -d option